src/studio/screens/console.c

```diff
@@ -926,7 +929,20 @@ static void onLoadCommandConfirmed(Console* console)
                 }
                 else printError(console, "\nfile not found");
 #else
-                printError(console, "\ncart loading error");
+                if(tic_project_ext(name)) {
+                    printError(console, "\nproject loading error");
+                    printFront(console, "\nThis version only supports binary cartridges (GIF, PNG, or TIC).");
+                    printLine(console);
+                    printFront(console, "\nTIC-80 ");
+                    consolePrint(console,"PRO",tic_color_light_blue);
+                    printFront(console, " is needed for projects.");
```

Comment:
Thanks for looking into this.

It seems like this message is conflating "projects" and "text files". I guess within the code those are synonymous, but to an end user who has never read the implementation, there is no way to understand this.

The message should use the word "text" rather than "projects" because that's the term that the user will understand.

Reply:
I see your point, but I don't love that either.  I think perhaps we just repeat the extension of the file?

- `...is needed for LUA projects.`
- `...is needed for WREN projects.`
- `...is needed for JS projects.`

Is that clearer? And if projects is still confusing I support we could replace it with `files`.

Reply:
I don't think it's really any clearer, because it's possible to have Lua, Wren, JS, etc games which are stored in binary carts or in text files. If you specifically want to avoid the word "text" for some reason then `needed for .lua files` would be better than putting the language in all-caps, since LUA is not an acronym. But saying "text files" is definitely clearer.

Reply:
I also vote for the "text files" or "text carts"